### PR TITLE
[TE] fix rootcause chart color bug

### PR DESCRIPTION
**Pull request by justYves**
### What's new: 
- fix the color bug: this was caused by ember-c3 using the color string, instead of the mapping. Adding a mapping helper fixed the issue

### Test: 
Visual Sanity check

### CODE REVIEW

File: thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart/component.js [hunk 1]
```diff
@@ -354,7 +364,7 @@ export default Ember.Component.extend({
       return {
         timestamps: [entities[urn].start, entities[urn].end || entities[urn].start],
         values: [val, val],
-        color: entities[urn].color,
+        color: getColor(entities[urn].color),
```
Comment (apucher) [inline thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart/component.js]: please also delete the code in timeseries-chart that applies a CSS class from the color string (lines 73-75)

File: thirdeye/thirdeye-frontend/app/helpers/utils.js [hunk 2]
```diff
@@ -228,4 +249,24 @@ export function findLabelMapping(label, config) {
...
+  appendTail,
+  hasPrefix,
+  filterPrefix,
+  toBaselineRange,
+  toFilters,
+  toFilterMap,
+  findLabelMapping,
+  toMetricLabel,
+  toColor,
+  getColor});
```
Comment (apucher) [inline thirdeye/thirdeye-frontend/app/helpers/utils.js]: line break

File: thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart/component.js [hunk 3]
```diff
@@ -2,7 +2,17 @@ import Ember from 'ember';
...
+  getColor,
+  toBaselineUrn,
+  toMetricUrn,
+  filterPrefix,
+  hasPrefix,
+  stripTail,
+  toBaselineRange,
+  toFilters,
+  toMetricLabel
+} from 'thirdeye-frontend/helpers/utils';
```
Comment (apucher) [inline thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart/component.js]: i wonder if there's a better way of doing this throughout our project

Comment (justYves) [reply thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart/component.js]: We could also import the whole utils as 'utils' and use them in the code as utils.toFilters etc...

File: thirdeye/thirdeye-frontend/app/helpers/utils.js [hunk 4]
```diff
@@ -51,6 +63,15 @@ export function checkStatus(response, mode = 'get', recoverBlank = false) {
   }
 }
 
+/**
+ * Converts the color string into its rgba cunterpart
```
Comment (apucher) [inline thirdeye/thirdeye-frontend/app/helpers/utils.js]: nice typo. lets delete the method if its not used - we can keep the mapping const

Comment (justYves) [reply thirdeye/thirdeye-frontend/app/helpers/utils.js]: Oops haha

**Pull request merged**